examples/speaker_tasks/diarization/neural_diarizer/e2e_diarize_speech.py

```diff
@@ -82,6 +82,7 @@ class DiarizationConfig:
     no_der: bool = False
     out_rttm_dir: Optional[str] = None
     save_preds_tensors: bool = False
+    precision: str = "bf16"  # 32, bf16
```

Comment:
Let's also add the bf16-mixed option and maybe add a small comment about possible gains expected with bf16 training/inference?